setup.cfg

```diff
@@ -16,16 +16,19 @@ python_requires = >= 3.7
 install_requires =
     datalad >= 0.12.0
     jsmin
+    whoosh
```

Comment:
Some additional dependencies still need to be added here. Functionality is not mandatory, but without them the respective extractors would not work.

'PyYAML',  # very optional
'mutagen>=1.36',  # audio metadata
'exifread',  # EXIF metadata
'python-xmp-toolkit',  # XMP metadata, also requires 'exempi' to be available locally
'Pillow',  # generic image metadata

Reply:
There would also be a system dependency on libxmp, but it seems that has not been installed in any of the -core CIs too, hence these tests never ran (recently?), so maybe not worth reintroducing them now -- if they would fail, nobody will fix them anyways.

Reply:
Dependecies are added to `setup.cfg`.

Reply:
Dependecies were added to `setup.cfg`